orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/ecs/EcsServerGroupCreator.groovy

```diff
@@ -57,6 +84,23 @@ class EcsServerGroupCreator implements ServerGroupCreator, DeploymentDetailsAwar
     if (operation.useTaskDefinitionArtifact) {
       if (operation.taskDefinitionArtifact) {
         operation.resolvedTaskDefinitionArtifact = getTaskDefArtifact(stage, operation.taskDefinitionArtifact)
+        if (operation.evaluateTaskDefinitionArtifactExpressions) {
+          Iterable<Object> rawArtifact =
+              retrySupport.retry(
+                  fetchAndParseArtifact(operation.resolvedTaskDefinitionArtifact), 10, Duration.ofMillis(200), true)
```

Comment:
is there precedence for this retry config (`10, Duration.ofMillis(200), true`) elsewhere in Orca? what is it for similar manifest evaluations?

Reply:
Yes, it is similar for manifest evaluations.

Reply:
Link? (i believe you, but for future reference)

Reply:
https://github.com/spinnaker/orca/blob/76a9d1e4887ded5c5da952d459ec494111874cf8/orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestEvaluator.java#L131